app/Services/GitHub/GitHubSyncService.php

```
@@ -48,41 +52,41 @@ public function synchronize(bool $force = false): SyncResult
      */
     public function synchronizeOrganizations(bool $force = false): Collection
```

PHPMD (2.15.0)

The method synchronizeOrganizations has a boolean flag argument $force, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)